pages/common/cowsay.md

```diff
@@ -18,3 +18,7 @@
 - Print a stoned thinking ASCII cow:
 
 `cowthink -s "I'm just a cow, not a great thinker ..."`
+
+- Use text from standard input for the balloon:
```

Comment:
I think you can make this the 2nd example.

Reply:
I was wondering about moving it up... will do :)

Reply:
@mebeim - Did you commit your changes ? I don't see them.

Reply:
@agnivade oh sorry, I committed without adding any change :laughing: should be ok now.